src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2088,6 +2088,8 @@ internal enum ErrorCode
         WRN_ObsoleteMembersShouldNotBeRequired = 9042,
         ERR_RefReturningPropertiesCannotBeRequired = 9043,
 
+        ERR_LineSpanDirectiveRequiresSpace = 9028,
```

Comment:
>ERR_LineSpanDirectiveRequiresSpace = 9028, [](http://example.com/codeflow?start=8&length=42)

Consider moving to line 2074.

Reply:
Not sure how I got the right number, but in the wrong place... ;-)